io.github.mgerhardy.vengi.yml

```diff
@@ -0,0 +1,64 @@
+app-id: io.github.mgerhardy.vengi
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: vengi-voxedit
+rename-icon: voxedit-icon
+rename-desktop-file: vengi-voxedit.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
```

Comment:
This seems to use SDL 2, are you sure Wayland doesn't work?

Reply:
Are you sure it works?

Reply:
No, that's why I'm asking. I can try it.

Reply:
Wayland seems to work fine if I enable the Wayland socket and set X11 to be a fallback.

Reply:
Okay, thanks.